platformio.ini

```diff
@@ -9,10 +9,24 @@
 ; http://docs.platformio.org/page/projectconf.html
 
 [platformio]
-env_default = d1_mini
+env_default = d1_mini_pro
```

Comment:
we are not going to change the default here

Reply:
Sure, no problem. I have now most of ispindels with a pro board so I've changed this to default in my config.